src/definitions/modules/calendar.js

```diff
@@ -1300,10 +1304,10 @@ $.fn.calendar.settings = {
 
         //time with ':'
         for (i = 0; i < numbers.length; i++) {
-          var number = numbers[i];
```

Comment:
why removed `var`?
It seems that the variable `numbers` defined in `header` function scope, but no `number` defined, so it might be leaked into window (global) scop.e

Reply:
I moved the "number" (as well as "word") var declaration to the top of the function so it still has function scope.

      var words = text.split(settings.regExp.dateWords);
      var numbers = text.split(settings.regExp.dateNumbers);
      var words = text.split(settings.regExp.dateWords), word; //<-- Moved declaration here
      var numbers = text.split(settings.regExp.dateNumbers), number;  //<-- Moved declaration here

I removed it here, because its being unnecessarily redefined on each for loop iteration (micro optimation)